src/deepsparse/yolov8/README.md

```diff
@@ -51,7 +51,7 @@ This creates a `model.onnx` file, in the directory of your `weights` (e.g. `runs
 DeepSparse’s performance can be pushed even further by optimizing the model for inference. DeepSparse is built to take advantage of models that have been optimized with weight pruning 
 and quantization—techniques that dramatically shrink the required compute without dropping accuracy. Through our One-Shot optimization methods, which will be made available in an upcoming 
 product called Sparsify, we have produced YOLOv8s and YOLOv8n ONNX models that have been quantized to INT8 while maintaining at least 99% of the original FP32 mAP@0.5. 
```

Comment:
Does this claim still holds (in the context of the sparsezoo pointer): 
```
we have produced YOLOv8s and YOLOv8n ONNX models that have been quantized to INT8 while maintaining at least 99% of the original FP32 mAP@0.5
```
just making sure, we are saying nothing but truth and truth only